### PR TITLE
content: publish zephyr kconfig article

### PR DESCRIPTION
**Pull request by gminn**
- Adjusted grammar and punctuation for clarity
- Adjusted some spacing found by a markdown linter --> should look at adding this to a pre-commit hook!

### CODE REVIEW

File: _posts/2024-01-24-practical_zephyr_kconfig.md [hunk 1]
```diff
@@ -166,17 +165,16 @@ Message in a bottle.
...
 
 > **Notice:** The command line output of `west build` also shows the order in which _Kconfig_ merges the configuration files. We'll see this in the section about [build types and alternative Kconfig files](#build-types-and-alternative-kconfig-files).
 
-When debugging _Kconfig_ settings, it can be helpful to have a look at the generated output. All _Kconfig_ symbols are merged into a single `zephyr/.config` file located in the build directory, in our case `build/zephyr/.config`. There, we find the following setting:
+When debugging _Kconfig_ settings, it can be helpful to have a look at the generated output. All _Kconfig_ symbols are merged into a single `zephyr/.config` file located in the build directory, in our case, `build/zephyr/.config`. There, we find the following setting:
 
 `build/zephyr/.config`
+
```
Comment (gminn) [inline _posts/2024-01-24-practical_zephyr_kconfig.md:177]: Note this won't add a new line, it just conforms to a markdown linting rule for ``` braces to be surrounded by new lines

**Comment by gminn** [comment]
@lmapii I've applied a few grammar tweaks and formatting adjustments. Let me know what you think!

File: _posts/2024-01-24-practical_zephyr_kconfig.md (hunk 1 above)
Comment (lmapii) [reply _posts/2024-01-24-practical_zephyr_kconfig.md:177]: I was just using this to write the filename before code blocks. The block following this then uses the correct code block with the corresponding format for its contents.

**Comment by lmapii** [comment]
Looks good, thank you for the adjustments!

File: _posts/2024-01-24-practical_zephyr_kconfig.md (hunk 1 above)
Comment (gminn) [reply _posts/2024-01-24-practical_zephyr_kconfig.md:177]: I actually really like you add the filename before the code block! It makes the code blocks easier to understand.

Comment (lmapii) [reply _posts/2024-01-24-practical_zephyr_kconfig.md:177]: Aaaah now I get it: Your markdown linter complained that there was no newline before the code block and you added this newline to silence it (and this newly added newline doesn't actually create an empty new line - like a tongue twister). I'm a bit slow today, sorry.

Comment (gminn) [reply _posts/2024-01-24-practical_zephyr_kconfig.md:177]: Yep, exactly!

Comment (gminn) [reply _posts/2024-01-24-practical_zephyr_kconfig.md:177]: My comment probably could have used more precise language

**Review by bahildebrand — APPROVED** [review]
Another great article. Great work @lmapii !

**Comment by lmapii** [comment]
Very happy this is moving along, thank you so much for your time!

**Pull request merged**